lib/ui/painting.dart

```diff
@@ -824,37 +824,73 @@ enum FilterQuality {
 
 /// Styles to use for line endings.
 ///
-/// See [Paint.strokeCap].
+/// See also:
+///
+/// * [Paint.strokeCap] for how this value is used.
+/// * [StrokeJoin] for the different kinds of line segment joins.
 // These enum values must be kept in sync with SkPaint::Cap.
 enum StrokeCap {
   /// Begin and end contours with a flat edge and no extension.
+  ///
+  /// ![Butt Cap Diagram](https://flutter.github.io/assets-for-api-docs/assets/dart-ui/butt_cap.png)
+  ///
+  /// The cap is zero-width here, so there is nothing to highlight. Compare to
```

Comment:
For someone who is reading this out of context, without seeing the other enum values, this doesn't really make sense. What does "highlight' mean in this context? I'd just drop the sentence entirely.